scripts/download-ip2location-databases.js

```diff
@@ -70,12 +17,31 @@ const databases = {
 };
 
 async function main() {
+	if (!process.env.PN_ACT_CONFIG_IP2LOCATION_TOKEN) {
+		// * Optional
+		return;
+	}
+
 	for (const name in databases) {
 		const database = databases[name];
-		const directory = await unzipper.Open.url(request, `https://www.ip2location.com/download/?token=${process.env.PN_ACT_CONFIG_IP2LOCATION_TOKEN}&file=${name}`);
-		const file = directory.files.find(file => file.path === database.file_name);
-		const content = await file.buffer();
-		fs.writeFileSync(database.save_path, content);
+		const response = await fetch(`https://www.ip2location.com/download/?token=${process.env.PN_ACT_CONFIG_IP2LOCATION_TOKEN}&file=${name}`);
+		const arrayBuffer = await response.arrayBuffer();
+		const buffer = Buffer.from(arrayBuffer);
+		const zip = await yauzl.fromBuffer(buffer);
+
+		try {
+			for await (const entry of zip) {
+				console.log(entry.filename);
```

Comment:
Was this just a debug log? We should add some context to it if we intend to keep it - like "Processing zip file: X"

Reply:
It was a leftover, my bad